database/index.js

```diff
@@ -0,0 +1,63 @@
+const { Client } = require('pg');
+const Promise = require('bluebird');
+
+const client = new Client({
+  user: process.env.LENA_ORDERS_DB_USER || 'lena',
+  host: process.env.LENA_ORDERS_DB_HOST || 'localhost',
+  database: process.env.LENA_ORDERS_DB_data || 'transactions',
+  port: process.env.LENA_ORDERS_DB_PORT || 5432,
+});
+
+client.connect();
+
+const createOrder = (userId, shippingAddress) => {
+  const queryStr = `
+    INSERT INTO orders
+    (user_id, shipping_address, delivery_status, created_at)
+    VALUES
+    ($1, $2, 'created', $3)
+  `;
+  const currDate = new Date().toISOString().slice(0, 10);
+  const queryArgs = [userId, shippingAddress, currDate];
+  return new Promise((resolve, reject) => {
+    client.query(queryStr, queryArgs, (err, res) => {
+      if (err) {
```

Comment:
not a big deal, but i guess ternary operators would do here as well?

Reply:
I think this is good. Probably shouldn't use ternary operators to make function calls.